wiki/Hit_object/Juice_stream/en.md

```diff
@@ -6,6 +6,20 @@ stub: true
 
 *See also: [Score](/wiki/Score)*
 
-**Juice streams** are an element in [osu!catch](/wiki/Game_Modes/osu!catch) which include both drops and droplets. Drops give a score of 100, equivalent to slider ticks in [osu!](/wiki/Game_Modes/osu!), while droplets give a score of 10, equivalent to the slider tick in osu!.
+**Juice streams** are an element in [osu!catch](/wiki/Game_Modes/osu!catch) which include both drops and droplets. Juice streams get generated when a [slider](/wiki/Hit_object/Slider) is placed in the editor.
 
-<!-- TODO: Add links -->
+## Drop
+
+*Drops* are medium-sized coloured [hit objects](/wiki/Hit_object) present in [osu!catch](/wiki/Game_mode/osu!catch) [beatmaps](/wiki/Beatmap). Drops are equivalent to slider ticks in [osu!](/wiki/Game_Modes/osu!). A [hyperfruit](/wiki/Hit_object/Hyperfruit) will be generated on a drop once the next [fruit](/wiki/Hit_object/Fruit) or drop is uncatchable by normal dashing.
+
+Successfully catching a drop will earn the player a [score](/wiki/Score) of 100, increase the [combo](/wiki/Beatmapping/Combo) by 1, provide a small boost to the [health bar](/wiki/Glossary/Health_bar), and is treated as 100 in the result screen. If the player fails to catch a drop it will result in a [combo](/wiki/Beatmapping/Combo) break and a loss in [health](/wiki/Beatmapping/Health).
```

Comment:
```suggestion
Successfully catching a drop will earn the player a [score](/wiki/Score) of 100, increase the [combo](/wiki/Beatmapping/Combo) by 1, provide a small boost to the [health bar](/wiki/Glossary/Health_bar), and is treated as 100 in the result screen. If the player fails to catch a drop it will result in a [combobreak](/wiki/Glossary/Combobreak) and a loss in [health](/wiki/Beatmapping/Health).
```